Makefile

```diff
@@ -330,7 +330,7 @@ c2cgeoportal/version.py: gen_current_version
 
 .PHONY: gen_current_version
 gen_current_version:
-	@echo "# Copyright (c) 2019, Camptocamp SA" > c2cgeoportal/version.py.new
+	@echo "# Copyright (c) 2018-2019, Camptocamp SA" > c2cgeoportal/version.py.new
```

Comment:
Why this?

Reply:
The change shown by GitHub is confusing, in the current status it is this:
https://github.com/camptocamp/c2cgeoportal/blob/09fc819169adcbfeb36460ce4e88c4bd6ce06c54/Makefile#L333
So I am actually changing it from "2018" to "2018-2019"
(in a first commit, I simply changed "2018" to "2019", then I realized it makes more sense to change it to "2018-2019")

Reply:
For me not a file generated in 2019 should have only 2019 in the copyright notice... (it's what we do in start of 2018 :-) )

Reply:
That's fine for me. I changed it back to 2019.